### DISCUSSION TITLE
Will nextcord be abandoned?

### DISCUSSION THREAD

**Discussion by VaibhavSys** [nextcord/nextcord#523] [Q&A] [answered]
Now that discord.py is back, will nextcord be abandoned? I hope not.

**Comment by alentoghostflame** [comment, marked as answer]
Top pin in General on the discord server:
> Epic (Repo owner): Nextcord is not dead. We are working hard to get nextcordv3 out, and are making good progress. We hope you stay with us even though Danny has returned.
> 
> Now why should you use nextcord over dpy?
> We are working on cleaning up the internals in v3, allowing you to do tons of crazy things with extensions! This will also make the code much faster (see announcements )

**Reply by VaibhavSys** [reply]
Thanks alot! I coudn't access the nextcord discord server so I asked here.

**Reply by MaskDuck** [reply]
[invite for nextcord server here](https://discord.gg/nextcord) or maybe you were banned ig

**Reply by VaibhavSys** [reply]
> [invite for nextcord server here](https://discord.gg/nextcord) or maybe you were banned ig

Yes, I was tempbanned.
